sources/packages/backend/libs/sims-db/src/entities/student-restriction.model.ts

```diff
@@ -48,4 +57,54 @@ export class StudentRestriction extends BaseRestrictionModel {
     },
   )
   applicationRestrictionBypasses?: ApplicationRestrictionBypass[];
+
+  /**
+   * Date when the restriction was resolved.
+   */
+  @Column({
+    name: "resolved_at",
+    type: "timestamptz",
+    nullable: true,
+  })
+  resolvedAt?: Date;
+
+  /**
+   * User who resolved the restriction.
+   */
+  @ManyToOne(() => User, { nullable: true })
+  @JoinColumn({
+    name: "resolved_by",
+    referencedColumnName: "id",
```

Comment:
Can we use the id from the constant here following other places?

<img width="435" height="39" alt="image" src="https://github.com/user-attachments/assets/c0b43b0e-7c29-4a4e-9b95-8a5baafc03ff" />